src/App.tsx

```diff
@@ -1,223 +1,83 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+  useMemo,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Filter } from './types/Filter';
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
+import { Header } from './components/Header/Header';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(Filter.All);
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
```

Comment:
move it all to Header component. You need to add ref newTodoField to input (input will be on focus when you open page)
<img width="310" alt="Screenshot 2022-10-06 at 22 31 15" src="https://user-images.githubusercontent.com/47606379/194402358-dfa41e4c-b760-44ad-9771-2dad2d8b1042.png">